src/components/ContentRender/plugins_bac/remark-custom-variable.ts

```
@@ -1,187 +1,192 @@
+          .split(SEPARATOR_GLOBAL)
+          .map(text => text.trim())
+          .filter(text => text.length > 0),
+        placeholder: undefined,
+      };
 
     case FormatType.SINGLE_BUTTON:
       // ?[%{{variable}} button]
-      const buttonText = match[2].trim()
+      const buttonText = match[2].trim();
       return {
         variableName,
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)